frontend/src/scenes/paths/pathsLogic.ts

```diff
@@ -160,6 +169,16 @@ export const pathsLogic = kea<pathsLogicType<PathNode, PathResult>>({
                 }
             }
         },
+        openPersonsModal: ({ path_start_key, path_end_key, path_dropoff_key }) => {
+            personsModalLogic.actions.loadPeople({
+                action: 'session', // Not necessary - make optional
+                label: 'Person',
+                date_from: '',
+                date_to: '',
+                filters: { ...values.filter, path_start_key, path_end_key, path_dropoff_key },
+                saveOriginal: true, // TODO: what does this really do? Pagination help?
```

Comment:
@liyiy  any ideas? I traced the code a bit but this didn't make much sense to me

Reply:
This was being used for persons modal user search, but not sure if that's something we support still lol